packages/webpack-extraction-plugin/src/babelPluginStripGriffelRuntime.ts

```diff
@@ -79,6 +115,59 @@ export const babelPluginStripGriffelRuntime = declare<
                 );
               }
 
+              argumentPaths[1].traverse({
+                TemplateLiteral(literalPath) {
+                  const expressionPath = literalPath.get('expressions.0');
+
+                  if (Array.isArray(expressionPath) || !expressionPath.isIdentifier()) {
+                    throw literalPath.buildCodeFrameError(
+                      [
+                        'A template literal with an imported asset should contain an expression statement.',
+                        'Please report a bug (https://github.com/microsoft/griffel/issues) if this error happens',
+                      ].join(' '),
+                    );
+                  }
+
+                  const expressionName = expressionPath.node.name;
+                  const expressionBinding = literalPath.scope.getBinding(expressionName);
+
+                  if (typeof expressionBinding === 'undefined') {
+                    throw expressionPath.buildCodeFrameError(
+                      [
+                        'Failed to resolve a binding in a scope for an identifier.',
+                        'Please report a bug (https://github.com/microsoft/griffel/issues) if this error happens',
+                      ].join(' '),
+                    );
+                  }
+
+                  const importDeclarationPath = expressionBinding.path.findParent(p =>
+                    p.isImportDeclaration(),
+                  ) as NodePath<t.ImportDeclaration> | null;
+
+                  if (importDeclarationPath === null) {
+                    throw expressionBinding.path.buildCodeFrameError(
+                      [
+                        'Failed to resolve an import for the identifier.',
+                        'Please report a bug (https://github.com/microsoft/griffel/issues) if this error happens',
+                      ].join(' '),
+                    );
+                  }
+
+                  expressionPath.replaceWith(
+                    t.stringLiteral(
+                      // When imports are inlined, we need to adjust the relative paths inside url(..) expressions
+                      // to allow css-loader resolve an imported asset properly
+                      transformUrl(
+                        state.filename!,
+                        options.resourceDirectory!,
+                        importDeclarationPath.get('source').node.value,
+                      ),
+                    ),
+                  );
+                  importDeclarationPath.remove();
```

Comment:
why remove the import decalartion ? wouldn't it also get stripped by bundlers ?

Reply:
Technically these is no reason, but as these imports are used only for a specific declaration I decided to remove it to be a bit nicer 🐱